.cirrus.yml

```diff
@@ -169,11 +169,11 @@ build_task:
               CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
               # ID for re-use of build output
               CI_DESIRED_RUNTIME: crun
-              #- env: &priorfedora_envvars
-              #DISTRO_NV: ${PRIOR_FEDORA_NAME}
-              #VM_IMAGE_NAME: ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
-              #CTR_FQIN: ${PRIOR_FEDORA_CONTAINER_FQIN}
-              #CI_DESIRED_RUNTIME: crun
+        #- env: &priorfedora_envvars
+        #DISTRO_NV: ${PRIOR_FEDORA_NAME}
+        #VM_IMAGE_NAME: ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
+        #CTR_FQIN: ${PRIOR_FEDORA_CONTAINER_FQIN}
+        #CI_DESIRED_RUNTIME: crun
```

Comment:
I realize this is minor, but are these whitespace changes absolutely necessary? They clutter the diff, and I'm pretty sure that when PRIOR_FEDORA gets added back, it will be more difficult to get the right magic yaml indentation because these changes make it less obvious what to do. Given how critical even the tiniest whitespace is for yaml, I'd really like to see these whitespace changes reverted. Unless there's a reason for them? (the commit message does not give one.)

Reply:
> Given how critical even the tiniest whitespace is for yaml,

The previous commenting of these sections **added** incorrect whitespace. This change makes it correct, such that removing the `#` will put the sections in the proper place.  I'll re-submit with a separate commit for this.

Reply:
Oh, nevermind, I already made this a separate commit.  I'll clarify the message re: whitespace.